src/locales/en.json

```diff
@@ -461,6 +461,13 @@
       "futureShiftsTitle": "Scheduled shifts",
       "futureShiftsHint": "All shifts scheduled for the future.",
       "tagsNotes": "Tags/Notes"
+    },
+    "warnings": {
+      "selectedDateIsHoliday": "The selected date is a bank holiday. You can only clock shifts of type bank holday here.",
```

Comment:
hol**i**day

ggf. den shift type in Anführungszeichen setzen (of type "bank holiday")